### DISCUSSION TITLE
No motion generated while running omni_python examples/isaac_sim/motion_gen_reacher.py

### DISCUSSION THREAD

**Discussion by jayaramreddy10** [NVlabs/curobo#49] [Software Q&A] [answered]
Hi, I was running  omni_python examples/isaac_sim/motion_gen_reacher.py for motion generation. I am able to see the rendered franka panda arm along with red block which I am able to control but I am not able to see the end effector converging to red block. There is no movement of the arm. Am I missing something to get the motion? 

![no_motion_curobo_isaac_sim](https://github.com/NVlabs/curobo/assets/111297422/0c2ff7ef-c31d-45f5-98da-5f8240c15c11)

**Comment by balakumar-s** [comment, marked as answer]
You need to click the play button on the left.

https://curobo.org/source/getting_started/2b_isaacsim_examples.html#isaac-sim-basics
